packages/scaffold/src/partials/w3m-connecting-wc-browser/index.ts

```diff
@@ -38,7 +38,8 @@ export class W3mConnectingWcBrowser extends W3mConnectingWidget {
       } else if (injectedConnector) {
         await ConnectionController.connectExternal(injectedConnector)
       }
-      ModalController.close()
+      // ModalController.close()
```

Comment:
I think this is for testing purposes only atm, we should always add that info within comment as well.